lib/Elastica/Search.php

```diff
@@ -22,6 +22,7 @@ class Search
     const OPTION_SIZE = 'size';
     const OPTION_SCROLL = 'scroll';
     const OPTION_SCROLL_ID = 'scroll_id';
+    const OPTION_QUERY_CACHE = 'query_cache';
```

Comment:
In case you have boot2docker running, it is quite simple: 
```
make run RUN="phpunit -c ./test lib/Elastica/Test/SearchTest.php"
```
This runs the SearchTest.php file. The first time will take a little bit longer as it must fetch all the docker images.